packages/react-composites/src/composites/CallWithChatComposite/components/DialpadModal.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { useState } from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { useMemo } from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { Dialpad, DialpadStyles, _DrawerMenu, _DrawerMenuItemProps, _DrawerSurface } from '@internal/react-components';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { IButtonStyles, IModalStyles, Modal, Stack, useTheme, Text, IconButton, PrimaryButton } from '@fluentui/react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { Call20Regular } from '@fluentui/react-icons';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { drawerContainerStyles } from '../styles/CallWithChatCompositeStyles';
+
+/** @private */
+export interface DialpadModalStrings {
+  dialpadModalAriaLabel: string;
+  DialpadModalTitle: string;
+  closeModalButtonAriaLabel: string;
+  callButtonLabel: string;
+}
+
+/** @private */
+export interface DialpadModalProps {
+  isMobile: boolean;
+  isModalOpen: boolean;
+  strings: DialpadModalStrings;
+  hideModal: () => void;
+}
+
+/** @private */
+export const DialpadModal = (props: DialpadModalProps): JSX.Element => {
+  /* @conditional-compile-remove(PeoplePaneDropdown) */
+  const { strings, isMobile, isModalOpen, hideModal } = props;
+  /* @conditional-compile-remove(PeoplePaneDropdown) */
+  const [textfieldInput, setTextfieldInput] = useState('');
+  /* @conditional-compile-remove(PeoplePaneDropdown) */
+  const theme = useTheme();
+  /* @conditional-compile-remove(PeoplePaneDropdown) */
+  const onLightDismissTriggered = (): void => hideModal();
+  /* @conditional-compile-remove(PeoplePaneDropdown) */
+  const onChange = (input: string): void => {
```

Comment:
nit: again this can just be `onChange = setTextFieldInput` (or better still, simply pass `setTextFieldInput` as the callback below.

Reply:
nit: again this can just be `onChange = setTextFieldInput` (or )